tests/benchmarks/mbedtls/prj.conf

```diff
@@ -12,15 +12,8 @@ CONFIG_MBEDTLS_DEBUG_LEVEL=4
 
 # TLS configuration
 CONFIG_MBEDTLS=y
-CONFIG_MBEDTLS_BUILTIN=y
 CONFIG_MBEDTLS_ENABLE_HEAP=y
 CONFIG_MBEDTLS_HEAP_SIZE=64000
 CONFIG_MBEDTLS_SSL_MAX_CONTENT_LEN=2048
 
-CONFIG_MBEDTLS_TLS_VERSION_1_2=y
-CONFIG_MBEDTLS_KEY_EXCHANGE_ALL_ENABLED=y
-CONFIG_MBEDTLS_CIPHER_ALL_ENABLED=y
-CONFIG_MBEDTLS_ECP_ALL_ENABLED=y
-CONFIG_MBEDTLS_MAC_ALL_ENABLED=y
-CONFIG_MBEDTLS_GENPRIME_ENABLED=y
-CONFIG_MBEDTLS_HMAC_DRBG_ENABLED=y
```

Comment:
Those configs were introduced to make sure all of the algorithms supported by mbedTLS are tested. If you want to use the benchmark from the upstream, perhaps you should provide similar in nrf_security.

Reply:
Removed these. No longer needed to do